tests/index.js

```diff
@@ -13,15 +13,13 @@ test('standardMarkdown', function (t) {
     // console.error(JSON.stringify(results, null, 2))
 
     t.comment('dirty fixture')
-    t.equal(results.length, 6, 'returns six linting errors')
+    t.equal(results.length, 5, 'returns six linting errors')
 
-    t.equal(results[0].message, "'foo' is defined but never used", 'finds errors')
```

Comment:
This test is no longer an "error" has we have disabled the `no-unused-vars` rule